docs/users.md

```diff
@@ -0,0 +1,78 @@
+# Users
+
+This is a collection of user methods. 
```

Comment:
We should add some mention about these being convenience wrappers around the Eventbrite V3 API [User endpoints](https://www.eventbrite.com/platform/api/#/reference/user)

Reply:
Updated. We should probably do a scan of all of the docs in this repo to make sure they still point to something useful with the launch of the new docs site.